sonar-helm-for-iac/helm-imports.go

```diff
@@ -268,3 +280,65 @@ func required(warningMessage string, value interface{}) (interface{}, error) {
 func emptyText(_ string) string {
 	return ""
 }
+
+func toYaml(input interface{}) string {
```

Comment:
The functions all look good!
I'm not 100% familiar with the importing logic of go, but can't we just import this file (https://github.com/helm/helm/blob/c6d2f6dba1dd5fd2e417167b4964e5b0a8bb0beb/pkg/engine/funcs.go) and use the functions from there? Than we wouldn't need to reimplement the logic.

Reply:
We don't have Helm as our dependency